packages/holocron/__tests__/HolocronErrorBoundary.spec.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { mount } from 'enzyme';
+import HolocronErrorBoundary, { withHolocronErrorBoundary } from '../src/HolocronErrorBoundary';
+
+describe('HolocronErrorBoundary', () => {
+  let consoleErrorSpy;
+  let mockError;
+  let CharacterWelcome;
+  let Character;
+
+  const { NODE_ENV } = process.env;
+
+  beforeAll(() => {
+    consoleErrorSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+    mockError = new Error(
+      'These are not the droids you are looking for.'
+    );
+
+    // eslint-disable-next-line react/prop-types
+    Character = ({ name }) => {
+      switch (name) {
+        case 'R2-D2':
+        case 'C-3PO':
+        case 'BB-8':
+          throw mockError;
+
+        default:
+          return <p>{`Welcome ${name}`}</p>;
+      }
+    };
+
+    // eslint-disable-next-line react/prop-types
+    CharacterWelcome = ({ name }) => (
+      <div>
+        <p>{`Looking up character ${name}`}</p>
+        <Character name={name} />
+      </div>
+    );
+  });
+
+  afterAll(() => {
+    consoleErrorSpy.mockRestore();
+  });
+
+  afterEach(() => {
+    process.env.NODE_ENV = NODE_ENV;
+  });
+
+  it('Renders the child component if there is no error', () => {
+    const wrapper = mount(
+      <HolocronErrorBoundary>
+        <CharacterWelcome name="Chewbacca" />
+      </HolocronErrorBoundary>
+    );
+
+    const CharacterWelcomeWithHolocronErrorBoundary = withHolocronErrorBoundary(CharacterWelcome);
+    const wrapperWithHolocronErrorBoundary = mount(
+      <CharacterWelcomeWithHolocronErrorBoundary name="Chewbacca" />
+    );
+
+    expect(wrapper.state().error).toBe(null);
+
+    expect(
+      wrapper.contains(<CharacterWelcome name="Chewbacca" />)
+    ).toBe(true);
+
+    expect(
+      wrapperWithHolocronErrorBoundary.find(HolocronErrorBoundary).instance().state.error
+    ).toBe(null);
+
+    expect(
+      wrapperWithHolocronErrorBoundary.contains(
+        <CharacterWelcome name="Chewbacca" />
+      )
+    ).toBe(true);
+  });
+
+  it('Sets its state t an error state and renders the error message', () => {
```

Comment:
```suggestion
  it('Sets its state to an error state and renders the error message', () => {
```

Reply:
I dont see any assertions on the render as the spec suggests

Reply:
Taken care of in b6e3011